keras_hub/src/layers/modeling/rotary_embedding.py

```diff
@@ -138,6 +152,43 @@ def _get_inverse_freq(self, rotary_dim):
             ops.cast(rotary_dim, "float32"),
         )
         inverse_freq = 1.0 / (self.max_wavelength**freq_range)
+
+        # For llama3.1
+        if all(
+            x is not None
+            for x in (
+                self.low_freq_factor,
```

Comment:
these always need to be set together? probably stick a check in init that these are all none or all set and raise a value error otherwise